src/CountriesArray.php

```diff
@@ -62,6 +62,7 @@ class CountriesArray{
         "CM" => array( 'alpha2'=>'CM', 'alpha3'=>'CMR', 'num'=>'120', 'isd'=> '231', "name" => "Cameroon", "continent" => "Africa"),
         "CA" => array( 'alpha2'=>'CA', 'alpha3'=>'CAN', 'num'=>'124', 'isd'=> '1', "name" => "Canada", "continent" => "North America"),
         "CV" => array( 'alpha2'=>'CV', 'alpha3'=>'CPV', 'num'=>'132', 'isd'=> '238', "name" => "Cape Verde", "continent" => "Africa"),
+        "BQ" => array( 'alpha2'=>'BQ', 'alpha3'=>'BES', 'num'=>'535', 'isd'=> '599', "name" => "Caribbean Netherlands", "continent" => "North America"),
```

Comment:
BQ is assigned to Bonaire, Sint Eustatius and Saba as per ISO list at https://www.iso.org/obp/ui/#search